cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -1443,13 +1423,7 @@ impl ManagedEnvironment {
     ) -> Result<PullResult, ManagedEnvironmentError> {
         // Check whether the local checkout is in sync with the current generation
         // before potentially updating generations and resetting the local checkout.
-        let remote = self
-            .generations()
-            .writable(flox.temp_dir.clone())
-            .map_err(ManagedEnvironmentError::CreateFloxmetaDir)?
-            .get_current_generation()
-            .map_err(ManagedEnvironmentError::CreateGenerationFiles)?;
-
+        let remote = self.generations();
```

Comment:
nit: would be more consistent to rename this
```suggestion
        let generations = self.generations();
```